Platon/backend/app/follow_system/views.py

```diff
@@ -1,10 +1,250 @@
-from flask import Response
+from flask import make_response, jsonify, request
 from flask_restplus import Resource
+from flask import current_app as app
 
-class FollowAPI(Resource):
-    def get(self):
-        return Response("Platon API is under development!!",200)
+from app import api, db
+from app.follow_system.forms import GetFollowingsForm, GetFollowersForm, GetFollowRequestsForm, \
+    SendFollowRequestsForm, AcceptFollowRequestsForm, RejectFollowRequestsForm
+from app.follow_system.forms import get_followings_parser, get_followers_parser, get_follow_requests_parser, send_follow_requests_parser, \
+    accept_follow_requests_parser, reject_follow_requests_parser
+from app.follow_system.models import Follow, FollowRequests
+from app.auth_system.views import login_required, generate_token
+
+import datetime
+from dateutil import parser
+from hashlib import sha256
+from functools import wraps
+
+
+# Private Account Restrictions not implemented.
```

Comment:
I implemented a decorator for this purpose. You can use it after PR is merged.

Reply:
Thanks for reminding.